mod_cluster/extension/src/main/java/org/wildfly/extension/mod_cluster/ModClusterConfigurationServiceBuilder.java

```diff
@@ -210,9 +210,12 @@ public ServiceName getServiceName() {
         optionalString(LOAD_BALANCING_GROUP.resolveModelAttribute(context, model)).ifPresent(group -> builder.node().setLoadBalancingGroup(group));
 
         optionalList(PROXIES.resolveModelAttribute(context, model)).ifPresent(
-                refs -> refs.stream()
-                        .map(ModelNode::asString)
-                        .forEach(ref -> this.outboundSocketBindings.add(new InjectedValueDependency<>(CommonUnaryRequirement.OUTBOUND_SOCKET_BINDING.getServiceName(context, ref), OutboundSocketBinding.class)))
+                refs -> {
```

Comment:
This preserves a lambda while the goal of the PR is to reduce the usage.

Reply:
True, though the goal of this change isn't to remove _all_ lambdas, just ones that can be done automatically.  Nevertheless, this code can be simplified further I think so I'll see what I can do.